econml/dml.py

```diff
@@ -866,7 +867,7 @@ def __init__(self,
                          random_state=random_state)
 
 
-class ForestDMLCateEstimator(NonParamDMLCateEstimator):
+class ForestDMLCateEstimator(NonParamDMLCateEstimator, ForestModelFinalCateEstimatorMixin):
```

Comment:
Generally speaking I believe the mixin should come first in the set of inherited classes to avoid having method lookup happen in an unexpected order.  It may not matter at the moment in this case because the mixin's logic never calls into anythingn from `super` but I think it should be switched around in case changes are made in the future.

Reply:
Done